blog/2021-07-06-Blog-Rohit-Property-Based-Testing.md

```diff
@@ -23,25 +23,26 @@ My work involved writing "Property-based tests for tremor-script" and some of th
 
 ### New concepts I learned specific to my work
 
+<!-- alex ignore his -->
 - Erlang and Rust
     - My work mostly revolved around Erlang and a little Rust and I was completely new to this ecosystem, it didn't help to not find much resources or actively accessible community for Erlang.
     - I took this as a challenge and went through [various resources to learn Erlang](https://github.com/diru1100/learn_erlang), functional programming in general and I was able to see why this Language was involved to do the task at hand, [my mentor](https://twitter.com/heinz_gies) is very passionate about Erlang and shared his thought-process, experience which helped me broaden my knowledge and how to approach any concept while learning something completely new.
 
 <!--truncate-->
 
 - Tremor-script
-    - It is an interpreted expression-oriented language designed for the filtering, extraction, transformation and streaming of structured data in a stream or event-based processing system which is explicitly turing incomplete used to easily write programs specific to Tremor use-cases.
+    - It is an interpreted expression-oriented language designed for the filtering, extraction, transformation and streaming of structured data in a stream or event-based processing system which is explicitly turing incomplete used to write programs specific to Tremor use-cases.
     - It is written using Rust but tested using Erlang
 - Property based testing
     - We have seen various types of testing approaches like Unit testing, Integration testing, End2End testing etc.
     - The purpose of tests is to check if our code is failing anywhere and test the same with various inputs.
-    - Fig 1 shows different types of tests to easily understand which features are specialised in what.
+    - Fig 1 shows different types of tests to understand which features are specialised in what.
```

Comment:
How about  ... tests to *help* understand...?

Reply:
good idea :)